cmd/ingester/app/flags.go

```diff
@@ -48,6 +48,24 @@ const (
 	SuffixParallelism = ".parallelism"
 	// SuffixHTTPPort is a suffix for the HTTP port
 	SuffixHTTPPort = ".http-port"
+	// SuffixAuthentication for enable or disable authentication when connect to kafka cluster.
+	SuffixAuthentication = ".authentication"
+	// KerberosPrefix for Kerberos configuration options
+	KerberosPrefix = ".kerberos"
+	// SuffixKerberosServiceName is the suffix for Kerberos service name
+	SuffixKerberosServiceName = ".serviceName"
```

Comment:
Instead of camel case, I think most flags use hyphens (e.g. service-name). Same applies to `useKeyTab` and `kerberosConfig`.